google/cloud/spanner_v1/_helpers.py

```diff
@@ -207,6 +217,12 @@ def _parse_value_pb(value_pb, field_type):
     :type field_type: :class:`~google.cloud.spanner_v1.types.Type`
     :param field_type: type code for the value
 
+    :type field_name: str
```

Comment:
Add optional in the documentation

Reply:
As mentioned in [comment](https://github.com/googleapis/python-spanner/pull/909#discussion_r1135549179), this is not an optional field.